tests/test_client.py

```diff
@@ -808,6 +808,10 @@ def create_refund(self, start_date, payment_response=None):
 
         refund = self._client.refunds.refund_payment(body)
         if refund.is_error():
+            if retryable and any([err.get('code') == "VERSION_MISMATCH" for err in refund.errors]):
```

Comment:
Can we do this via backoff instead?

Reply:
We can, I can try to implement that instead.

Reply:
You still have the recursive retry pattern.